pkg/apis/kops/validation/instancegroup_test.go

```diff
@@ -294,3 +298,65 @@ func TestIGCloudLabelIsIGName(t *testing.T) {
 		testErrors(t, g.label, errs, g.expected)
 	}
 }
+
+func TestInstanceMetadataOptions(t *testing.T) {
```

Comment:
I think these should be part of the AWS tests, so moved to that file.

Reply:
alright will do so.